core/server/services/mega/mega.js

```diff
@@ -423,11 +417,8 @@ async function getEmailMemberRows({emailModel, memberSegment, options}) {
     const knexOptions = _.pick(options, ['transacting', 'forUpdate']);
     const filterOptions = Object.assign({}, knexOptions);
 
-    let newsletter = null;
-    if (labsService.isSet('multipleNewsletters')) {
-        newsletter = await emailModel.related('newsletter').fetch(Object.assign({}, {require: false}, _.pick(options, ['transacting'])));
-    }
-    const recipientFilter = transformEmailRecipientFilter(emailModel.get('recipient_filter'), {errorProperty: 'recipient_filter'}, newsletter);
+    const newsletter = await emailModel.related('newsletter').fetch(Object.assign({}, {require: true}, _.pick(options, ['transacting'])));
```

Comment:
Does this need the same hack as above?

Reply:
Not that I noticed. Main problem is that I cannot reproduce it in tests 😬

Reply:
I'll add it to be sure + gonna continue to investigate the issue